### ISSUE TITLE
Unable to package for win64

### ISSUE THREAD

**Issue by grinkers** [ncsoft/Unreal.js#41]
Hello. First off, thanks for making such a great plugin publicly available!

I'm currently having trouble packaging the Examples project to create a standalone win64 executable. I followed the directions for Building and have had no problems building after creating an empty c++ MyActor to make it a c++ project. The only other change I've made is changing the default map for the Game and Editor to HelloSpringy. Everything seems to run fine within the Editor, but when I try to package I get an "Error Unknown Cook Failure" (see attached PackageOutputLog.txt for the full output and PackageCookLog.txt).

If I select File -> "Cook Content for Windows" I get a successful cook (DebugCookLog.txt) and then am able to build and run Development Win64 with the Visual Studio 2015 Debugger.

Any help would be greatly appreciated!

[PackageOutputLog.txt](https://github.com/ncsoft/Unreal.js/files/170393/PackageOutputLog.txt)
[PackageCookLog.txt](https://github.com/ncsoft/Unreal.js/files/170394/PackageCookLog.txt)
[DebugCookLog.txt](https://github.com/ncsoft/Unreal.js/files/170395/DebugCookLog.txt)

**Comment by nakosung** [comment]
Please check latest commit, I just fixed that problem. :)

**Comment by grinkers** [comment]
The package was successful!

However when I try to load the executable it's just a blank level. Are there further steps for packaging or moving the scripts over?

Thanks for your time!

**Comment by nakosung** [comment]
1. Copy `Plugins/UnrealJS/Content/Scripts` into `/Game/Content/Scripts`.
2. Add `/Game/Content/Scripts` folder into 'non-asset directories to copy' in package settings.

**Comment by nakosung** [comment]
Everything goes well?

**Comment by grinkers** [comment]
Success! Thank you very much.

Sorry for not replying immediately, I got distracted with trying to figure out how to get the scripts to correctly load while also in a .pak. I was doing this more to learn how this plugin works than needing to put it in a pak any time soon.

As far as I got, if I package the scripts folder with "Additional Non-Asset Directories to Package", I am able to read the contents of a .js file in the .pak with this code:
`FString FullPath = FPaths::GameDir() / "Content/Scripts/helloWorld.js";`

I tried adding a function in V8Module.cpp to add it to Paths with `return FPaths::GameDir() / "Content/Scripts/";` , but no luck.

Anyway, thanks again for your help and the plugin.
